react-debugging.html

```diff
@@ -65,22 +65,27 @@
         // No need to change anything ABOVE this line ^
 
         return (
-          <h1>Draw a Card</h1>
-          <button onClick={}>
+        <>
+        <h1>Draw a Card</h1>
+        
+          <button onClick={drawCard}>
             Click to Draw a Card
           </button>
-          <button onclick={shuffle}>
+          <button onClick={shuffle}>
             Return Cards and Shuffle Deck
           </button>
-          <div style={handStyles}>
+          <div style={handStyles}></div>
```

Comment:
Great job finding the bugs! 🐛 You actually do want the div right before the JSX fragment closing tag, otherwise it will mess with your styling.